### DISCUSSION TITLE
I need to adjust the eyelashes as nodes using the alpha value, but I can't adjust the transparency well.

### DISCUSSION THREAD

**Discussion by lenama5543** [bnpr/Malt#592] [Q&A]
I've watched all the videos on YouTube, but there's something I still don't know. I need to adjust the eyelashes as nodes using the alpha value, but I can't adjust the transparency well. What should I do?

**Comment by Psi25** [comment]
did you use vec4 and adjusted the 4th value?

**Comment by Psi25** [comment]
![image](https://github.com/user-attachments/assets/2fa7e2f2-aafb-489c-a430-d902061f489f)

don't forget to turn on transparency at the materials tab 

![image](https://github.com/user-attachments/assets/f9339e58-634d-431e-8035-7a710fe30fb5)

![image](https://github.com/user-attachments/assets/4db6da6a-27cb-4f7c-a074-db9336f7c446)

**Reply by lenama5543** [reply]
thanks!!!!!!
